unicodetools/data/ucd/dev/PropList.txt

```diff
@@ -1134,6 +1137,8 @@ AADD          ; Extender # Lm       TAI VIET SYMBOL SAM
 AAF3..AAF4    ; Extender # Lm   [2] MEETEI MAYEK SYLLABLE REPETITION MARK..MEETEI MAYEK WORD REPETITION MARK
 FF70          ; Extender # Lm       HALFWIDTH KATAKANA-HIRAGANA PROLONGED SOUND MARK
 10781..10782  ; Extender # Lm   [2] MODIFIER LETTER SUPERSCRIPT TRIANGULAR COLON..MODIFIER LETTER SUPERSCRIPT HALF TRIANGULAR COLON
+10D4E         ; Extender # Lo       GARAY VOWEL LENGTH MARK
+10D6A         ; Extender # Mn       GARAY CONSONANT GEMINATION MARK
```

Comment:
I would expect the U+10D6F GARAY REDUPLICATION MARK to also be an Extender, like U+3005 IDEOGRAPHIC ITERATION MARK.

Reply:
Hm, it reduplicates a word rather than a character though.

But  U+3031〱 and U+3032 〲 are also extenders, so this should indeed be an extender. (Assuming their function as described in https://en.wikipedia.org/wiki/Iteration_mark is accurate enough.)